infra/terraform/hash/hash_application/api.tf

```
@@ -76,7 +76,7 @@ locals {
     mountPoints = []
     volumesFrom = []
     dependsOn = [
-      { condition = "SUCCESS", containerName = local.api_migration_container_def.name },
+      # { condition = "SUCCESS", containerName = local.api_migration_container_def.name },
```

This was for testing only
```suggestion
      { condition = "SUCCESS", containerName = local.api_migration_container_def.name },
```